### PR TITLE
Add a ci directory containing scripts for integration with a CI system

### PR DESCRIPTION
**Pull request by mestery**
This commit adds some wrapper scripts around the ovn-scale-test ansible which
are meant to be used for integration with a continuous integration system.

Signed-off-by: Kyle Mestery mestery@mestery.com

### CODE REVIEW

File: ci/scale-run.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+pushd $OVN_SCALE_REPO_NAME
+sudo /usr/local/bin/ansible-playbook -i $OVN_DOCKER_HOSTS ansible/site.yml -e @$OVN_DOCKER_VARS \
+     --extra-vars "ovs_repo=$OVS_REPO" --extra-vars "ovs_branch=$OVS_BRANCH" -e action=deploy
+popd
+
+# Create the rally deployment
+docker exec ovn-rally rally-ovs deployment create --file /root/rally-ovn/ovn-multihost-deployment.json --name ovn-multihost
+
+# Register the emulated sandboxes in the rally database
+docker exec ovn-rally rally-ovs task start /root/rally-ovn/workload/create-sandbox-$OVN_RALLY_HOSTNAME.json
```
Comment (huikang) [inline ci/scale-run.sh:44]: I think this should be within a for loop for all the nodes in the [emulation-hosts] of the inventory file.

File: ci/scale-run.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+# Build the docker containers
+pushd $OVN_SCALE_REPO_NAME
+cd ansible/docker
+make
+popd
+
+# Deploy the containers
+pushd $OVN_SCALE_REPO_NAME
+sudo /usr/local/bin/ansible-playbook -i $OVN_DOCKER_HOSTS ansible/site.yml -e @$OVN_DOCKER_VARS \
```
Comment (huikang) [inline ci/scale-run.sh:32]: After executing the asnbile command, can we add some sanity check to ensure all emulated hypervisor are connected to the southbound db, e.g., by "docker exec southbound-database ovn-sbctl show"?

Comment (mestery) [inline ci/scale-run.sh:32]: Yes, good call!

**Comment by mestery** [comment]
I think this is ready to merge now.

**Comment by huikang** [comment]
LGTM

**Pull request merged**